packages/manager/modules/telecom-spare/src/modems/order/modems-order.scss

```diff
@@ -0,0 +1,7 @@
+.telecom-spare--modems-order-section {
+  margin-bottom: 5px;
+}
+
+.modems-order-section-content {
+  margin: 0 32px;
```

Comment:
This looks like random values applied. Can't we avoid those ?

Reply:
It's the same values than the .telecom-section-content class defined